examples/plugin/src/test/java/com/github/bsideup/liiklus/plugins/example/support/AbstractIntegrationTest.java

```diff
@@ -45,21 +51,36 @@ protected PublishReply publishRecord(String key, String value) {
         return client.publish(PublishRequest.newBuilder()
                 .setTopic(topic)
                 .setKey(ByteString.copyFromUtf8(key))
-                .setValue(ByteString.copyFromUtf8(value))
+                .setLiiklusEvent(
+                        LiiklusEvent.newBuilder()
+                                .setId(UUID.randomUUID().toString())
+                                .setType("com.example.event")
+                                .setSource("/tests")
+                                .setDataContentType("application/json")
+                                .putExtensions("comexampleextension1", "foo")
+                                .setData(ByteString.copyFromUtf8(value))
+                                .setTime(ZonedDateTime.now().format(DateTimeFormatter.ISO_OFFSET_DATE_TIME))
+                                .buildPartial()
+                )
```

Comment:
unrelated to this PR

Reply:
yeah, yeah, I've got it :) will split as well.